gulpfile.js

```diff
@@ -247,6 +250,23 @@ gulp.task("link-bugs", function (done) {
     exec(bin("linkify-bugs.sh"), [path.join(SOURCE_DIR, "_posts")], done);
 });
 
+gulp.task('lint', function() {
```

Comment:
Only when all linter errors are fixed, can we have build depend on lint. For now, it should be an independent task. We don't have jslint. I filed it as a separate feature to add, but it won't be in this PR.